src/pages/dashboard/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { useRouter } from 'next/router';
+import React from 'react';
+import PageNotFound from '../404';
+import NavBar from '@/components/navBar';
+import Searchbar from '@/components/Dashboard/Searchbar';
+
+const search = (query: string) => {
+    const searchValues = query.split(',');
+    console.log('Searching', searchValues);
```

Comment:
????

Reply:
This is a dummy function currently as the feature is broken down into small parts thus currently this function just console logs

will be updated in future PRs with functionality

Reply:
Also the feature is behind a feature flag so it is not directly accessible to users

Reply:
also why not take this function out in a different util file?

Reply:
done, created a new util